lib/routes/actionsController.js

```diff
@@ -113,7 +113,6 @@ function PostAction(req, resp) {
     } else {
         delay = config.slaveDelay;
     }
-    // TODO: Tricky ... To refactor?
```

Comment:
It seems to be a mark of pending technical debt. The refactor has not been done so I understand it should keep.